getlantern.org/static/css/main.css

```diff
@@ -456,6 +463,10 @@ section#faq {
   a.download-desktop {
     display: none;
   }
+  
+  p.version-number {
+	  display: none !important;
```

Comment:
I've used jQuery to hide/show `.version-number`, for hiding it in mobile view we need `!important` to make sure browser ignore inline style `display: block;` which jQuery added to element